.gitignore

```diff
@@ -63,6 +63,8 @@ nbproject/
 .idea
 cmake-build-*/*
 coverage-report/
+.direnv
```

Comment:
This should probably be in your global .gitignore instead of added to this one

Reply:
I added this here as that's the more standard way to use a flake but I understand some people might manually do `nix develop`